src/core/error/common_error.rs

```diff
@@ -255,6 +255,7 @@ pub enum CommonError {
     FailedToDeserializeJSONToValue {
```

Comment:
Would be nice to also add the message in the `#[error` on top

Reply:
Does this affect the size of the enum? I was afraid to add it.

Reply:
yes it does, but we can add it for now, and remove it once UniFFI explodes :D